### PR TITLE
add build

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://pypi.org/project/build/

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge/staged-recipes team. This is ready for review!

File: recipes/build/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/build-{{ version }}.tar.gz
+  sha256: 5c6ca6cba06ea9a51623ce3197f019db8925ef3f06bf6d983fe5dcb16fcd1f6e
+
+build:
+  number: 0
+  skip: true  # [py<=34]
```
Comment (chrisburr) [inline recipes/build/meta.yaml:14]: `noarch: python`?

Comment (chrisburr) [reply recipes/build/meta.yaml:14]: `importlib-metadata` is `noarch` so it could avoid having the selector

**Comment by isuruf** [comment]
I'm not a fan of the name `build`. Maybe we could do `python-build`?

File: recipes/build/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "build" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/build-{{ version }}.tar.gz
```
Comment (beckermr) [inline recipes/build/meta.yaml:9]: Are we sure this package is from the upstream source? I was checking on pypi and the project home linked to a fork of pypa/build.

**Review by beckermr — CHANGES_REQUESTED** [review]

File: recipes/build/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/build/meta.yaml:9]: It is a mess but yeah. It is like that.

**Comment by ocefpaf** [comment]
> I'm not a fan of the name `build`. Maybe we could do `python-build`?

Agreed. That is a terrible name. Worse than pep517.

File: recipes/build/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/build/meta.yaml:14]: Yep. Thanks!

**Comment by ocefpaf** [comment]
Ready for another round @conda-forge/staged-recipes.

File: recipes/build/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/build/meta.yaml]: BTW, this is the PyPA blessed package for pep517 builds.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**